app/views/changeActivity/ChangeActivityCYAView.scala.html

```diff
@@ -16,18 +16,29 @@
 
 @this(
     layout: templates.Layout,
-    govukSummaryList: GovukSummaryList
+    govukSummaryList: GovukSummaryList,
+    formHelper: FormWithCSRF,
+    govukButton: GovukButton
 )
 
-@(seqSummaryList: Seq[(String, SummaryList)])(implicit request: Request[_], messages: Messages)
+@(
+    alias: String,
+    returnPeriod: String,
+    seqSummaryList: Seq[(String, SummaryList)],
+    submitCall: Call
+)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("checkYourAnswers.title"))) {
+@layout(pageTitle = titleNoForm(messages("changeActivity.checkYourAnswers.title"))) {
 
-    <h1 class="govuk-heading-l">@messages("checkYourAnswers.heading")</h1>
+    <span class="govuk-caption-l" id="pre-header-caption"> @alias - @returnPeriod </span>
+    <h1 class="govuk-heading-l">@messages("changeActivity.checkYourAnswers.heading")</h1>
 
+    @for((heading, summaryList) <- seqSummaryList) {
+        <h2 class="govuk-heading-m">@messages(heading)</h2>
+        @govukSummaryList(summaryList)
+    }
 
-   @for((heading, summaryList) <- seqSummaryList) {
-     <h2 class="govuk-heading-m">@heading</h2>
-   @govukSummaryList(summaryList)
-}
+    @formHelper(action = submitCall, Symbol("autoComplete") -> "off") {
+        @govukButton(ButtonViewModel(messages("updateRegisteredDetails.checkYourAnswers.submit")))
+    }
 }
```

Comment:
See screenshot - for CYA - amount-produced = None, old service shows Use third party packagers but does not show Packaged in the UK.  
![Screenshot 2023-07-11 at 15 12 29](https://github.com/hmrc/soft-drinks-industry-levy-variations-frontend/assets/90277886/40216678-a455-439b-841e-5bdc10d2c0c6)

Reply:
I will investigate/fix this. The answers on those last three sections are possibly defaulting to No - so I'd need to solve that in addition if so.

Reply:
Fixed this